A3A/addons/core/functions/Revive/fn_respawn.sqf

```diff
@@ -17,4 +17,7 @@ if (captive _unit) then {
 };
 
 _unit setVariable ["respawning",false];
-_unit setDamage 1;
\ No newline at end of file
+_unit setDamage 1;
+
+// GAMERARMY: Reset the one-time use revive.
+_unit setVariable ["oneTimeRevive",false];
```

Comment:
This should also be reset when using the big orange box at HQ, but that might just call fn_respawn for all I know, I'm not actually sure what it does.

Let me know if this is the case, otherwise try to add it to the box.